chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -132,11 +124,11 @@
 rails_env = chef_env_to_rails_env[node.chef_environment]
 
 logrotate_app 'rails-wca' do
-  frequency WCA_LOGROTATE_FREQUENCY
-  maxsize WCA_LOGROTATE_MAXSIZE
-  size WCA_LOGROTATE_SIZE
-  rotate WCA_LOGROTATE_ROTATE
-  path "#{repo_root}/WcaOnRails/log/production.log"
+  path		"#{repo_root}/WcaOnRails/log/production.log"
+  size		"512M"
+  maxage	90
+  postrotate	"[ ! -f /home/cubing/worldcubeassociation.org/WcaOnRails/pids/unicorn.pid ] || kill -USR1 `cat /home/cubing/worldcubeassociation.org/WcaOnRails/pids/unicorn.pid`"
+  options	['nodelaycompress']
```

Comment:
Nit: For consistency with the rest of this file (and our codebase), let's not align things like this. One space should be consistent with what we do elsewhere.